### DISCUSSION TITLE
What is the purpose of Include this elemrnt in the index for taxonomy?

### DISCUSSION THREAD

**Discussion by liusming** [OrchardCMS/OrchardCore#12576] [Q&A]
Hi,

I have checked "Include this element in the index" to Taxonomy as well as the "Stored" and "Analyzed" checed for indexing as follwing picture
 
![Screen Shot 2022-10-06 at 10 40 09 AM](https://user-images.githubusercontent.com/4685718/194203322-ce36c29b-fbe3-4c0d-91e2-2613fe39662c.png)

Hower I cannot tell the difference between with and without "Stored" and "Analyzed" checked. It all the same with the query result as following.
 
![Screen Shot 2022-10-06 at 10 46 06 AM](https://user-images.githubusercontent.com/4685718/194203598-572bf7e1-8241-461d-9edd-12a150f4ba9b.png)

I would like to have the value as readable text for taxnomy to filter my contentutem but seem always return the HEX code from query result no matter "Stored" and "Analyzed" are checked or not. 

Please kindly advise how I can deal with that or anything missing. Thanks in advance.

**Comment by Skrypt** [comment]
A Taxonomy Term is a Content Item. So, you need to retrieve the Content Item related to this/these UID to get its ContentItem.DisplayText value.

**Reply by samuel-tw** [reply]
Thanks, I will try it.  Would you please kindly adivse what is difference between check and uncheck the "Stored" and "Analyzed" under the " Include this element in the index"?
